Traceback (most recent call last):
  File "/home/gopi/.local/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/usr/lib/python3/dist-packages/nbclient/client.py", line 1093, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/lib/python3/dist-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/usr/lib/python3/dist-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/usr/lib/python3/dist-packages/nbclient/client.py", line 559, in async_execute
    await self.async_execute_cell(
  File "/usr/lib/python3/dist-packages/nbclient/client.py", line 854, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/usr/lib/python3/dist-packages/nbclient/client.py", line 756, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from langchain_community.llms import Ollama

MODEL = "phi3:latest"

llm = Ollama(model=MODEL, temperature=0.5)

------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [0;32mIn[1], line 1[0m
[0;32m----> 1[0m [38;5;28;01mfrom[39;00m [38;5;21;01mlangchain_community[39;00m[38;5;21;01m.[39;00m[38;5;21;01mllms[39;00m [38;5;28;01mimport[39;00m Ollama
[1;32m      3[0m MODEL [38;5;241m=[39m [38;5;124m"[39m[38;5;124mphi3:latest[39m[38;5;124m"[39m
[1;32m      5[0m llm [38;5;241m=[39m Ollama(model[38;5;241m=[39mMODEL, temperature[38;5;241m=[39m[38;5;241m0.5[39m)

[0;31mModuleNotFoundError[0m: No module named 'langchain_community'
ModuleNotFoundError: No module named 'langchain_community'

